Mathlib/AlgebraicTopology/SimplicialSet/Presentable.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+module
+
+public import Mathlib.AlgebraicTopology.SimplicialSet.FiniteColimits
+public import Mathlib.AlgebraicTopology.SimplicialSet.FiniteProd
+public import Mathlib.AlgebraicTopology.SimplicialSet.RegularEpi
+public import Mathlib.CategoryTheory.EffectiveEpi.Coequalizer
+public import Mathlib.CategoryTheory.Presentable.Limits
+
+/-!
+# Finite simplicial sets are presentable
+
+In this file, we show that finite simplicial sets are `ℵ₀`-presentable,
+which will allow the use of the small object argument in `SSet`.
+
+-/
+
+@[expose] public section
+
+universe u
+
+open CategoryTheory Simplicial Limits Opposite
+
+namespace CategoryTheory
+
+end CategoryTheory
+
+namespace SSet
+
+attribute [local instance] Cardinal.fact_isRegular_aleph0
+
+namespace Finite
+
+instance (n : SimplexCategory) :
+    IsCardinalPresentable (stdSimplex.{u}.obj n) Cardinal.aleph0.{u} where
+  preservesColimitOfShape J _ _ := by
+    let e : coyoneda.obj (op (stdSimplex.{u}.obj n)) ≅ (evaluation _ _).obj (op n) :=
+      NatIso.ofComponents (fun X ↦ yonedaEquiv.toIso)
+    apply preservesColimitsOfShape_of_natIso e.symm
```

Comment:
I would have expected this proof to be 
```
inferInstanceAs <| IsCardinalPresentable (uliftYoneda.obj n) Cardinal.aleph0.{u}
```
but it seems that the file `CategoryTheory/Presentable/Presheaf.lean` only gives instance for the "free yonedas". There might be restrictions to put on the universes levels, but I think in the long run we want instances like this in general presheaves categories and not just the simplex category. Can you please look into it?